src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -16,7 +16,7 @@
             <input type="checkbox" class="custom-control-input" value="@contentItem.Id" name="itemIds" id="itemIds-@contentItem.Id">
             <label class="custom-control-label" for="itemIds-@contentItem.Id"></label>
         </div>
-        <a admin-for="@contentItem" asp-route-returnUrl="@FullRequestPath" /> - <small class="text-muted">@contentItem.ContentType</small>
+        <a admin-for="@contentItem" asp-route-returnUrl="@FullRequestPath" /> @if (contentItem.Content["ListPart"] != null){<span class="badge badge-info">List</span>} - <small class="text-muted">@contentItem.ContentType</small>
```

Comment:
This would have to be done by a driver, (there is a zone for `Meta`)

Reply:
Is there an advantage for that?